spark8t/cli/pyspark.py

```diff
@@ -38,10 +36,27 @@
     )
 
     if service_account is None:
-        raise ValueError("Service account provided does not exist.")
+        raise NoAccountFound()
```

Comment:
Why don't we just stop execution and simply return nothing i.e. empty result?

That would be consistent with the behavior of the API.